content/en/network_monitoring/devices/vpn_monitoring.md

```diff
@@ -0,0 +1,134 @@
+---
+title: VPN Monitoring
+description: Get started with monitoring your devices VPN tunnels.
+further_reading:
+  - link: "https://www.datadoghq.com/knowledge-center/network-monitoring/snmp-monitoring/"
+    tag: "Knowledge Center"
+    text: "SNMP Monitoring Overview"
+  - link: "https://www.datadoghq.com/blog/monitor-snmp-with-datadog/"
+    tag: "Blog"
+    text: "Monitor SNMP with Datadog"
+---
+
+## Overview
+
+VPN monitoring provides visibility into your devices VPN tunnels within Network Device Monitoring (NDM). This feature allows you to keep track of critical data and metrics to examine the health of your VPN tunnels.
+
+When configured, the SNMP check collects VPN tunnel data from your devices. This can be configured for individual as well as Autodiscovered devices.
```

Comment:
```suggestion
When configured, the SNMP check collects VPN tunnel data from your devices. This can be configured for [individual][4] as well as [Autodiscovered][5] devices.
```